esmvalcore/_task.py

```diff
@@ -264,12 +264,12 @@ def get_product_attributes(self) -> dict:
             for product in self.products
         }
 
-    def str(self):
+    def print_ancestors(self):
         """Return a nicely formatted description."""
         def _indent(txt):
-            return '\n'.join('\t' + line for line in txt.split('\n'))
+            return '\n'.join('  ' + line for line in txt.split('\n'))
```

Comment:
Might be better to use the indent function from here:
https://docs.python.org/3/library/textwrap.html#textwrap.indent

Reply:
Done